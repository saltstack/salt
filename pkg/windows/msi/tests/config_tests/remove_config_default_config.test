properties CONFIG_TYPE=Default START_MINION="" REMOVE_CONFIG=1
