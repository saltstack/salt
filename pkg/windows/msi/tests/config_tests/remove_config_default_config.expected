# This is the template from the msi store line 1/6
#master: salt
# This is the template from the msi store line 2/6
#id:
# This is the template from the msi store line 3/6
# This is the template from the msi store line 4/6
# This is the template from the msi store line 5/6
# This is the template from the msi store line 6/6
